extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -393,11 +390,6 @@ const actions = (state, action, data) => {
         ...state,
         [StateProperty.KEYBOARD_ACTIVE_STATE]: !!data,
       });
-    case Action.TOGGLE_SUPPORTED_BROWSER:
-      return /** @type {!State} */ ({
-        ...state,
-        [StateProperty.SUPPORTED_BROWSER_STATE]: !!data,
-      });
```

Comment:
It's not possible nor desirable to restore the unsupported browser layer after it's been removed. As such, it's unnecessary to allow toggle this state multiple times.